Library/brew.rb

```diff
@@ -50,6 +50,20 @@
   EOABORT
 end
 
+if OS.mac? && MacOS.version == "10.11" && MacOS::Xcode.version < "7.0"
```

Comment:
i.e., it doesn't really make sense for things like `brew --prefix` or `brew info` or `brew deps` to outright fail if Xcode is outdated. We already have pre-install checks for this situation.